.gitignore

```diff
@@ -0,0 +1,23 @@
+# Logs
```

Comment:
Probably don't need all of this I guess, but no harm!

Reply:
yh this just came with vite XD thought i would just leave it in there